homeassistant/components/mpchc/media_player.py

```diff
@@ -81,6 +81,7 @@ def update(self):
                 self._player_variables[var[0]] = var[1].lower()
         except requests.exceptions.RequestException:
             _LOGGER.error("Could not connect to MPC-HC at: %s", self._url)
+            self._player_variables = dict()
```

Comment:
Besides this, you should implement the `available` property. Make it return `False` if the connection is lost to indicate to Home Assistant that the entity is currently not available.

Reply:
Thank you. I changed the codes.